H2TauTau/python/proto/analyzers/JetAnalyzer.py

```diff
@@ -160,6 +163,15 @@ def process(self, event):
             self.counters.counter('jets').inc('at least 1 b jet')
             if len(event.cleanBJets) > 1:
                 self.counters.counter('jets').inc('at least 2 b jets')
+                
+        # save HTs
+        event.HT_allJets     = sum([jet.et() for jet in allJets          ])
```

Comment:
We usually define HT as sum pT in CMS, but this might be for a specific use case where ET is used?